### ISSUE TITLE
Zone name appearance & link behaviour

### ISSUE THREAD

**Issue by kadilier** [UoC-Radio/rastapank-site#6]
Στο κέιμενο πχ 

`Ακούτε Latin Jazz
από τους πρόποδες του κάμπου(s) στο Ηράκλειο` 

το όνομα της ζώνης θα ήταν πιο σωστό να έχει άλλο style (να πηγαίνει το μάτι κατευθείαν μιας και η άλλη ποληροφορία μένει πάντα ίδια). Υποθέτω μελλοντικά όταν στηθεί και η βάση από πίσω θα μπορεί να πηγαίνει σε link description της ζώνης, άλλα για τώρα ένα διαφορειτικό style θα μέτραγε.

**Comment by ggalan87** [comment]
Ναι, εμένα μ αρεσει ακριβως γιατί ειναι integrated μέσα στο text... αλλά μπορεί να μπει κατι... italics π.χ.?

**Comment by kadilier** [comment]
Και χρωμα ισως αντι style, καθε ζωνη και διαφορετικο (τυχαίο) :p
